src/core/tests/pass/serialization/deterministicity.cpp

```diff
@@ -8,31 +8,35 @@
 
 #include "common_test_utils/common_utils.hpp"
 #include "common_test_utils/file_utils.hpp"
+#include "openvino/opsets/opset1.hpp"
 #include "openvino/pass/serialize.hpp"
 #include "openvino/util/file_util.hpp"
 #include "read_ir.hpp"
 #include "util/test_common.hpp"
 
-class SerializationDeterministicityTest : public ov::test::TestsCommon {
+class DeterministicityCommon {
 protected:
-    std::string m_out_xml_path_1;
-    std::string m_out_bin_path_1;
-    std::string m_out_xml_path_2;
-    std::string m_out_bin_path_2;
+    std::string m_out_xml_path_1 = {};
+    std::string m_out_bin_path_1 = {};
+    std::string m_out_xml_path_2 = {};
+    std::string m_out_bin_path_2 = {};
+    std::string xmlFileName;
 
-    void SetUp() override {
+    void SetupFileNames() {
         std::string filePrefix = CommonTestUtils::generateTestFilePrefix();
         m_out_xml_path_1 = filePrefix + "1" + ".xml";
         m_out_bin_path_1 = filePrefix + "1" + ".bin";
         m_out_xml_path_2 = filePrefix + "2" + ".xml";
         m_out_bin_path_2 = filePrefix + "2" + ".bin";
+        xmlFileName = filePrefix + "_TestModel.xml";
```

Comment:
The field is used only in one test. It would be better to replace it from common part directly to the test